src/addrs.rs

```diff
@@ -470,10 +471,19 @@ impl BackendOrganizationFileLinkAddr {
                 organization_id.0,
                 workspace_id.0,
                 encrypted_path,
+                encrypted_timestamp,
             ),
         ))
     }
 
+    #[getter]
+    fn encrypted_timestamp<'py>(&'py self, python: Python<'py>) -> Option<&'py PyBytes> {
```

Comment:
Why `self` is bounded by `'py` lifetime ?

Reply:
You're right `self` does not need to be bound to the `'py` lifetime here